### ISSUE TITLE
Consolidate student's submit logic

### ISSUE THREAD

**Issue by guido-ferraro** [JuiceMind/JuiceMind#240]
Currently there's one endpoint to handle each `InteractiveBlock` submit request and one hook for each one. This should be consolidated into one hook and one endpoint since a large part of the behavior is similar. This will not only simplify the code base a lot, but it will also make easier [backend migration to typescript](https://github.com/JuiceMind/JuiceMind/issues/119), writing tests, debug, maintaining or extending the functionalities, etc.

**Comment by marklee45** [comment]
@guido-ferraro is this done?

**Comment by guido-ferraro** [comment]
@marklee45 no, but you can make this `sev5` or the lowest. It's a code improvement
